subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/classpath/impl/ClasspathFingerprintCompareStrategyTest.groovy

```diff
@@ -21,224 +21,142 @@ import com.google.common.collect.Iterables
 import org.gradle.internal.execution.history.changes.ClasspathCompareStrategy
 import org.gradle.internal.execution.history.changes.CollectingChangeVisitor
 import org.gradle.internal.execution.history.changes.DefaultFileChange
-import org.gradle.internal.execution.history.changes.FingerprintCompareStrategy
 import org.gradle.internal.execution.history.impl.SerializableFileCollectionFingerprint
 import org.gradle.internal.file.FileType
 import org.gradle.internal.fingerprint.FileSystemLocationFingerprint
 import org.gradle.internal.fingerprint.impl.DefaultFileSystemLocationFingerprint
 import org.gradle.internal.hash.HashCode
 import spock.lang.Specification
-import spock.lang.Unroll
 
 class ClasspathFingerprintCompareStrategyTest extends Specification {
 
     private static final CLASSPATH = ClasspathCompareStrategy.INSTANCE
 
-    @Unroll
-    def "empty snapshots (#strategy, include added: #includeAdded)"() {
+    def "empty snapshots"() {
         expect:
-        changes(strategy, includeAdded,
-            [:],
+        changes([:],
             [:]
```

Comment:
I think both `[:]` should still be on its their lines, as the point of the formatting is to align the new and the old state.

Reply:
Good point!